.github/workflows/sdk-typescript-publish.yml

```diff
@@ -5,7 +5,7 @@ on:
 jobs:
   publish:
     if: github.repository == 'dagger/dagger'
-    runs-on: "${{ github.repository == 'dagger/dagger' && 'dagger-v0-12-3-4c-nvme' || 'ubuntu-latest' }}"
+    runs-on: "${{ github.repository == 'dagger/dagger' && 'dagger-g2-v0-12-3-4c' || 'ubuntu-latest' }}"
```

Comment:
I think that this should be `od` so that it cannot get interrupted during publishing.